tsconfig.json

```diff
@@ -9,6 +9,10 @@
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
     "esModuleInterop": true,
+    "lib": [
+      "ES2019.Array",
+      "ES2020.Promise"
```

Comment:
Doesn't this require shim?

Reply:
no, 
Promise.allSettled is supported from node 12.9 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/allSettled
Array.flat is supported from node 11 https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat